app/src/main/java/com/goodinc/teamtrack/FragmentMoviesList.kt

```diff
@@ -0,0 +1,34 @@
+package com.goodinc.teamtrack
+
+import android.content.Context
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+
+class FragmentMoviesList : Fragment() {
```

Comment:
done

Reply:
are u sure?

Reply:
not..
done 20%:
companion object {
fun newInstance() : FragmentMoviesList = FragmentMoviesList()
}

 =)

Reply:
why **onDetach** is unnecessary?
```
override fun onDetach() {
        super.onDetach()
        _cardClickListener = null_
    }
```
cardClickListener - how it will be come _null_?

Reply:
done..